tools/sigh.js

```diff
@@ -180,7 +180,7 @@ async function lint(args) {
   if (options.fix) {
     extra.push('--fix');
   }
-  let jsSources = [...findProjectFiles(process.cwd(), fullPath => /\.js$/.test(fullPath))];
+  let jsSources = [...findProjectFiles(process.cwd(), fullPath => /\.js$/.test(fullPath) && !/\.min\.js$/.test(fullPath))];
```

Comment:
This should be part of `findProjectFiles` instead of here. But it should already skip any `build/` directories, perhaps that list of directories needs to be updated, or something renamed to `build/`.

Reply:
Oh, I see this is probably about `frame-htmltexture-component.min.js`, rather than an actual build artifact. Can we put that in a third_party/ directory and skip `third_party` in findProjectFiles?

Reply:
ah, it's an 3rd party library already minified: shell/components/a-frame/aframe-htmltexture-component.min.js. moved it to what i hope is the right place.

Reply:
just saw that, yes, shell/components/third_party makes sense. will change.